dojo.py

```diff
@@ -1,310 +1,248 @@
+
+    def reallocate_person(self, person_id, room_name):
+        print(Dojo.all_rooms)
+        if (person_id in Dojo.people_keys) and (room_name.upper() in Dojo.room_keys):
+            room_key = room_name.upper()
+            room = Dojo.app_session['room'][room_key]
+            person = Dojo.app_session['person'][person_id]
+            if (check_room_size(room)):
+                if room.purpose == 'office':
+                    if person.office:
+                        delete_person_from_room(
+                            person, Dojo.app_session['room'][person.office])
+                        person.office = add_person_to_room(person, room)
+                        print('{0} has been allocated the office {1}'.format(
+                            person.fname, room.name))
                     else:
-                        file
-
-
-
+                        person.office = add_person_to_room(person, room)
+                        print('{0} has been allocated the office {1}'.format(
+                            person.fname, room.name))
+                else:
+                    if person.role == 'fellow':
+                        if person.wants_accommodation == 'y':
+                            if person.living_space:
+                                delete_person_from_room(
+                                    person, Dojo.app_session['room'][
+                                        person.living_space])
+                                person.living_space = add_person_to_room(
+                                    person, room)
+                                print(
+                                    '{0} has been allocated\
+                                    the living space {1}'.
+                                    format(person.fname, room.name))
+                            else:
+                                person.living_space = add_person_to_room(
+                                    person, room)
+                                print(
+                                    '{0} has been allocated\
+                                    the living space {1}'.
+                                    format(person.fname, room.name))
+                    else:
+                        print('You cannot add {} a Staff to a living Space.'.
+                              format(person.fname))
 
-    def reallocate_person(self, person_name, room_name):
-        pass
+            else:
+                print('You cannot Reallocate to a full Room')
+        else:
+            print('Invalid Person Id or Room name')
 
     def load_people(self):
-        pass
+        file = open('people.txt')
+
+        for line in file.readlines():
+            values = line.split()
+            if len(values) == 3:
+                self.add_person(values[0], values[1], values[2])
+            else:
+                self.add_person(values[0], values[1], values[2], values[3])
```

Comment:
What happens if `len(values) = 2`. This would surely throw an error. You need to handle that